### PR TITLE
My week 5 at Microverse and my first capstone project.

### PR DESCRIPTION
**Pull request by eerapheal**
This is my HTML, CSS and JavaScript capstone project. The application incorporates the following components:
- An HTML markup file.
- A CSS stylesheet file.
- A JavaScript script file.
- A gitignore file.
- Linters install.
- A license has been added to this project.

## Learning objectives
- Understand how to use medium-fidelity wireframes to create a UI.
- Follow written requirements (e.g. user stories).
- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Use images and backgrounds to enhance the look of a website.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.
- Communicate technical concepts to other technical people.

Below is the link to the recorded video documenting the development process of my app.
[Presentation video](https://www.loom.com/share/4d388388155346bbba40019551dc2ff3)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  </header>
+  <!--  -->
+  <main>
+     <section id="home" class="first-sction">
+      <h3 class="hello">"Hello everyone"</h3>
+      <h1 class="ralph">RALPH03 IS STUDY PLATFORM <br>YOU CAN ACCESS ANYWHERE</h1>
+      <P class="first-p">Online education is the newly evolved form of education<br> where students study from their homes with the help <br> of computers and the internet. join us.</P>
+      <h3 class="date">2022.10.15(THUR) ~ (16) FRI</h3>
+        <P class="first-body">@National Museum of Korea, Art Center Nabi and More</P>
+    </section>
```
Comment (afanimartin) [inline index.html:56]: - Please move this section to the `about` page.

Comment (afanimartin) [inline index.html:56]: - Please move this section to the `about` page.

File: README.md [hunk 2]
```diff
@@ -1,75 +1,57 @@
...
 
-### Install
+## Website link: 
 
-### Usage
 
-### Run tests
+## Setup
+Clone this repo with:
+https://github.com/eerapheal/capstone-project1
```
Comment (afanimartin) [inline README.md]: - Please be more explicit with the instruction for cloning. Include the actual commands for cloning. This makes it easier for a beginner to get started with your project.

File: README.md [hunk 3]
```diff
@@ -1,75 +1,57 @@
...
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./LICENSE) licensed.
+License
+This project is Ralph licensed.
```
Comment (afanimartin) [inline README.md]: - Use the MIT license. It's the standard and well known in the industry.

File: README.md [hunk 4]
```diff
@@ -1,75 +1,57 @@
...
-
 ## 🤝 Contributing
-
 Contributions, issues, and feature requests are welcome!
-
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+## Deployment
+Github
```
Comment (afanimartin) [inline README.md]: - How does one deploy to Github? Please include precise instructions.

File: README.md [hunk 5]
```diff
@@ -1,75 +1,57 @@
...
-### Usage
 
-### Run tests
+## Setup
+Clone this repo with:
+https://github.com/eerapheal/capstone-project1
 
-### Deployment
+## Install
+npm install
```
Comment (afanimartin) [inline README.md:33]: - How do I open the website locally? Please include instructions.

File: README.md [hunk 6]
```diff
@@ -1,75 +1,57 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+## Getting starting
+This repository includes files with plain HTML CSS AND JAVASCRIPTS.
```
Comment (afanimartin) [inline README.md]: - This sounds more like a description of the project than a getting started instructions. Please update this.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>module one Capstone project.</title>
+</head>
+<body id="home">
+  <header>
+    <nav>
```
Comment (afanimartin) [inline index.html:13]: - Add a closing for this `nav` element. Every element should have a closing. It's a good practice.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @eerapheal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🎉 🎉 🎉 
- Nice work with the project. You followed the given design.

## Required Changes ♻️
_Check the comments under the review._
- Please create `about.html` file and add content specific to the page in it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (eerapheal) [reply index.html:58]: Hey,
Thanks for the review. this is for the first section page not the about page, the one for the about page is right there in the about section.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @eerapheal, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

- [ ] Great job for pushing your project this far. However, there're few things to work on before your project could be due for approval.
    -  It would be better to have a separate file for home page and about page. You can then reference to the about page inside your home page file like  this ` <a class="nav-link1" href="#about.html">About</a>`. This was stated by the previous CR, but is not yet implemented. Kindly consider it worth doing.
    - Kindly check the guiding [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) to find out the content of each of the two pages: home and about and fill them with their respective content.

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f574770523531743971623739534e464f74732f67697068792e676966

## Hey @eerapheal :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Descriptive PR Summary
:heavy_check_mark: Descriptive PR Title

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:
# Mandatory
- [ ] As the previous reviewer suggested that you should have a separate html file for your about page so when the user will click on About link on the navbar then your page will be redirected to the about page. About page is the second page if you can see in the given design in the requirements 😊 You can watch this [repository](https://github.com/HAMMAS-SALEEM/Pakistan-Watch-Week-2022) to get an idea 🤔
# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+    <h3>Feature Speakers</h3>
+  <hr class ="fa-skp-line">
+  </div>
+  <div id="speakersGrid">
+  </div>
+</section>
+  </main>
+  <script src="./script.js"></script>
+</body>
+</html>
```
Comment (Abenezer-Tilahun) [inline index.html:130]: - [ Optional  ] I recommend you to add "partner " and "footer" section for Home page Desktop Screen size just like [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).

The Design Template of Home page  Look likes this : 
Capture

Your Homepage Look Likes this : 
Capture

File: me.html [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+          online enrollments continue to grow at rates faster than for the broader student population and
+          institutes of higher education expect the rate of growth to continue increasing. Some of the key
+          findings: By far the most popular approach today is online learning. According to the Sloan Consortium,
+          online enrollments continue to grow at rates faster than for the broader student population and
+          institutes of higher education expect the rate of growth to continue increasing. Some of the key
+          findings: 
+        </p>
+        <P class="reach">
+          Please contact us about Ralph03 online study via email <span id="email">ekpenisiraphael@gmail.com</span>
+        </P>
```
Comment (Abenezer-Tilahun) [inline me.html:70]: - [ Optional ] Kindly work on the "About " page, Your Heading are not fully responsive for Desktop Screen size, Please try to adjust it to make Your website Fully Responsive. 

Your About Page Look Likes this  : 
Capture1

The Design About Page Look likes this : 
Capture

File: me.html [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+          <a class="nav-link" href="#contactt">Join</a>
+          <a class="nav-link" href="#contactt">Sponsor</a>
+          <a class="nav-link" href="#contactt">News</a>
+          <a class="nav-link" href="#contactt">CC Campaign</a>
+      </div>
+      <div id="ham" class="hamburger">
+        <div class="hamburger-bar"></div>
+        <div class="hamburger-bar"></div>
+        <div class="hamburger-bar"></div>
+      </div>
```
Comment (Abenezer-Tilahun) [inline me.html:44]: - [ optional ] For Mobile screen size I recommend you to add "Home" page link at the "About " page to redirected to "Home" page which helps to improve user interaction.

Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @eerapheal , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**